projects/cloud/app/frontend/stores/UserStore.ts

```diff
@@ -0,0 +1,21 @@
+import { MeQuery, MeDocument } from '@/graphql/types';
+import { ApolloClient } from '@apollo/client';
+import { makeAutoObservable, runInAction } from 'mobx';
+
+export default class UsersStore {
+  me: MeQuery['me'];
+  client: ApolloClient<object>;
+  constructor(client: ApolloClient<object>) {
+    this.client = client;
+    makeAutoObservable(this);
+  }
+
+  async load() {
+    const { data } = await this.client.query({
```

Comment:
How are errors propagated here?

Reply:
Errors are silenced here. I could destructure the `errors` propety and re-throw it, so it's eventually caught by `ErrorBoundary`.